bcda/auth/plugin/alpha.go

```diff
@@ -248,18 +247,37 @@ func revokeAccessTokenByID(tokenID uuid.UUID) error {
 	return db.Error
 }
 
-func (p *AlphaAuthPlugin) ValidateAccessToken(token string) error {
-	return errors.New("not yet implemented")
+func (p *AlphaAuthPlugin) ValidateAccessToken(tokenString string) error {
+	t, err := p.DecodeAccessToken(tokenString)
+	if err != nil {
+		return err
+	}
+
+	b := isBlacklisted(t)
+	if b {
+		return fmt.Errorf("blacklisted Token with ID: %v was rejected", t.Claims.(*AllClaims).ID)
+	}
+
+	return t.Claims.(*AllClaims).Valid()
```

Comment:
`t.Valid` returns a bool and is populated when you parse a token.  `Claims.Valid()` runs all of the standard verifications and returns an error.  It's a little nicer to have the error over just `false`

Reply:
i agree with you on the variable naming but it is [go convention](https://github.com/golang/go/wiki/CodeReviewComments#variable-names)
since go is statically typed, we can tell easily the type of any variable